cpu/stm32/Kconfig

```diff
@@ -31,8 +31,8 @@ config HAS_BOOTLOADER_STM32
         Indicates that the stm32 bootloader is being used.
 
 config ERROR_MODULES_CONFLICT
-    default "On STM32, the RTC and RTT map to the same hardware peripheral." if MODULE_PERIPH_RTC && MODULE_PERIPH_RTT
-    depends on CPU_STM32
+    default "On STM32F1, the RTC and RTT map to the same hardware peripheral." if MODULE_PERIPH_RTC && MODULE_PERIPH_RTT
```

Comment:
Should this rather be moved to `cpu/stm32/kconfigs/f1/Kconfig` ?

Reply:
Good idea!

Reply:
Moved